spec/controllers/users_controller_spec.rb

```diff
@@ -122,14 +121,13 @@
     end
   end
 
-  describe "DELETE #destroy" do
-    it "destroys the requested user" do
+  describe 'DELETE #destroy' do
+    it 'destroys the requested user' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.